src/common/localization/en-us.ts

```diff
@@ -160,11 +162,43 @@ export const english: IAppStrings = {
             tfRecords: "Tensorflow Records",
             tfPascalVoc: "Tensorflow Pascal VOC",
         },
+        messages: {
+            saveSuccess: "Successfully saved export settings",
+        },
     },
     activeLearning: {
         title: "Active Learning",
     },
     profile: {
         settings: "Profile Settings",
     },
+    errors: {
+        Unknown: {
```

Comment:
Reason for Pascal over camel? no opinion either way, would be good to standardize across the board though

Reply:
Agree, will convert to camelCase.